hybrid-helper.ts

```diff
@@ -1,27 +1,28 @@
 import { downgradeComponent, downgradeInjectable } from '@angular/upgrade/static';
 import { FactoryProvider } from '@angular/core';
+import * as angular from 'angular';
 
 export interface IComponentUpgradeOptions {
-  inputs?: string[],
-  outputs?: string[]
+  inputs?: string[];
+  outputs?: string[];
 }
 
 export interface IHybridHelper {
-  downgradeComponent(moduleName: string, componentSelector: string, componentClass: any, options?: IComponentUpgradeOptions): IHybridHelper,
-  downgradeProvider(moduleName: string, providerName: string, providerClass: any): IHybridHelper,
-  buildProviderForUpgrade(ng1Name: string, ng2Name?: string): FactoryProvider
+  downgradeComponent(moduleName: string, componentSelector: string, componentClass: any, options?: IComponentUpgradeOptions): IHybridHelper;
+  downgradeProvider(moduleName: string, providerName: string, providerClass: any): IHybridHelper;
+  buildProviderForUpgrade(ng1Name: string, ng2Name?: string): FactoryProvider;
 }
 
-export const HybridHelper: IHybridHelper {
+export const HybridHelper: IHybridHelper = {
 
-  downgradeComponent: (moduleName: string, componentName: string, componentClass: any, options?: IComponentUpgradeOptions): IHybridHelper => {
+  downgradeComponent: (moduleName: string, componentSelector: string, componentClass: any, options?: IComponentUpgradeOptions): IHybridHelper => {
     options = options || {};
     const inputs = options.inputs || [];
     const outputs = options.outputs || [];
     const component = componentClass;
 
-    angular.module(moduleName).directive(componentName, downgradeComponent({ 
-      component, inputs, outputs 
+    angular.module(moduleName).directive(componentSelector, downgradeComponent({
```

Comment:
It's not the componentSelector. Selector has a dash.
I prefer to be consistence with the AngularJS code.
Please revert this change and I will merge this pull request.

The definition of the directive function in the AngularJS code: 
`          /**
           * @ngdoc method
           * @name angular.Module#directive
           * @module ng
           * @param {string|Object} name Directive name, or an object map of directives where the
           *    keys are the names and the values are the factories.
           * @param {Function} directiveFactory Factory function for creating new instance of
           * directives.
           * @description
           * See {@link ng.$compileProvider#directive $compileProvider.directive()}.
           */
          directive: invokeLaterAndSetModuleName('$compileProvider', 'directive'),
`